catalog-rest-service/src/main/java/org/openmetadata/catalog/util/EntityUtil.java

```diff
@@ -445,4 +470,14 @@ public static String getLocalColumnName(String fqn) {
     localColumnName.append(s[s.length - 1]);
     return localColumnName.toString();
   }
+
+  public static Boolean toBoolean(Include include) {
```

Comment:
else is not required
we can reduce the indentation like so and yield the same result

```
  public static Boolean toBoolean(Include include) {
    if (include.equals(DELETED)) {
      return Boolean.TRUE;
    }
    if (include.equals(ALL)) {
      return null;
    }
    // "non-deleted"
    return Boolean.FALSE;
  }
```

Reply:
It's a matter of taste. I prefer to use `else` in this case.

Reply:
I'm closing for now. Just reopen it if needed.